web/src/recent_view_ui.ts

```diff
@@ -722,8 +722,8 @@ function format_conversation(conversation_data: ConversationData): ConversationC
                     name: people.get_display_full_name(user.id),
                     status_emoji_info: user_status.get_status_emoji(user.id),
                 }),
-            )
-            .sort();
+            );
+        rendered_pm_with_html.sort();
```

Comment:
What's the benefit of this approach over doing `.toSorted` for these? I feel like the latter is probably the better solution for most of these.

Reply:
`.sort` avoids allocating a new array and sorts in-place, so it’s marginally faster.

Reply:
Yeah I'm not convinced the benefit is worth it vs the downside of having a variable that's temporarily mis-described as sorted for some of these cases. But to the extent a different choice might be better, it's probably too marginal for me to feel tempted to suggest redoing it. I'll just merge.